src/s3/commands/downloadFileAs.ts

```diff
@@ -119,9 +120,17 @@ export async function downloadFileAsCommand(
     const sourcePath = readablePath(node)
 
     await telemetry.s3_downloadObject.run(async () => {
-        const saveLocation = await promptForSaveLocation(file.name, window)
+        let lastUsedDownloadPath: string | undefined
+        const storedPath = globals.context.globalState.get('workbench.explorer.downloadPath')
```

Comment:
Can we extract `'workbench.explorer.downloadPath'` to a top level variable?

Reply:
rather, let's put all of this logic in a `getDefaultDownloadPath()` function